web/src/config.json

```diff
@@ -1,3 +1,3 @@
 {
-  
+  "backUrl": "http://192.168.99.100:31002"
```

Comment:
You shouldn't commit this file.

Reply:
I just deleted this file from the branch. Is it okay?